src/main/resources/com/nirima/jenkins/plugins/docker/DockerManagementServer/docker-manage.js

```diff
@@ -0,0 +1,18 @@
+function stop(theId) {
+    var input = document.getElementById('stopId');
+    if (input != null) {
+        input.value = theId;
+    }
+
+    var form = document.getElementById('control');
+    form.submit();
+}
```

Comment:
This code previously lived in `src/main/webapp/js/docker-manage.js` and has been moved without alteration. It has been moved here so it can be consumed through a Stapler adjunct.